lemur/auth/views.py

```diff
@@ -379,12 +379,10 @@ def post(self):
             verify_cert=verify_cert
         )
 
-        jwks_url = current_app.config.get('PING_JWKS_URL')
+        jwks_url = current_app.config.get('OAUTH2_JWKS_URL')
```

Comment:
Hey! Sorry for sitting on this for a while. This is a breaking change for us, and potentially for other Lemur users. Would you mind checking for both of these configuration values and issuing a warning if they are using an older value? Still allow the older value to be used for now if it is the only one configured, and we will deprecate it later.